tests/parser.cpp

```diff
@@ -228,6 +228,18 @@ Program
  kprobe:f
   builtin: args
 )");
+
+  test("uprobe:xxxx:f { __builtin_elf_is_exe }", R"(
```

Comment:
You can drop these tests from the Parser because `__builtin_elf_is_exe` is an ident and not a builtin token

Reply:
Removed, thanks.